src/screens/Home/index.tsx

```diff
@@ -1,16 +1,60 @@
-import React from 'react';
-import i18next from 'i18next';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useQuery } from 'react-query';
+
+import defaultCover from 'assets/book-cover.png';
+import { getBookList } from 'services/bookService';
 
-import logo from './assets/logo.svg';
 import styles from './styles.module.scss';
 
+interface BookInfo {
+  author: string;
+  createdAt: string;
+  editor: string;
+  genre: string;
+  id: number;
+  imageUrl: string;
+  title: string;
+  updatedAt: string;
+  year: string;
+}
+
+interface Books {
+  count: number;
+  currentPage: number;
+  nextPage?: number;
+  page: BookInfo[];
+  totalCount: number;
+  totalPages: number;
+}
```

Comment:
those interfaces should be on a file `types.ts` if you only use them on home screen level, or in the general types.ts if more general